roles/rsyslog/tasks/inputs/files/main.yml

```diff
@@ -15,9 +16,10 @@
         type: input
         state: "{{ item.state | d('present') }}"
         sections:
-          - options: |-
-              input(type="imfile" file="{{ item.input_log_path }}" tag="{{ item.name }}")
-              {{ lookup('template', 'input_template.j2') }}
+          - options: 'input(type="imfile"
+            file="{{ item.input_log_path }}" tag="{{ item.name }}")
+
```

Comment:
blank line here?

Reply:
I thought otherwise the lines above and below are concatenated... (not tried, though)

Reply:
it depends - what do you want the output to look like?
```
input(type="imfile" file="{{ item.input_log_path }}" tag="{{ item.name }}")

template begins here
...
```
or like this:
```
input(type="imfile" file="{{ item.input_log_path }}" tag="{{ item.name }}")
template begins here
...
```
or does it matter if the `input` is wrapped in the output?
```
input(type="imfile" file="{{ item.input_log_path }}"
         tag="{{ item.name }}")
template begins here
...
```
If it is the last one, then use `|-`:
```yaml
options: |-
  input(type="imfile" file="{{ item.input_log_path }}"
           tag="{{ item.name }}")
  {{ lookup("template", "input_template.j2") }}
```

Reply:
I'd like to have the first or the second one. Not the last one.
So, I think this works.
```
options: >-
  input(type="imfile" file="{{ item.input_log_path }}"
           tag="{{ item.name }}")
  {{ lookup("template", "input_template.j2") }}
```